### ISSUE TITLE
Twitch Ads

### ISSUE THREAD

**Issue by Osyruu** [pixeltris/TwitchAdSolutions#13]
Don't work again.

**Comment by pixeltris** [comment]
That's not very descriptive...

Which solution are you having problems with? What's your browser version? Are you using the userscript or ublock variant? Which country are you seeing ads from?

**Comment by Osyruu** [comment]
I see ads, I have the last version of the version, i'm from France

**Comment by pixeltris** [comment]
Which solution are you having problems with? Which browser are you using (i.e chrome, firefox, opera, etc)? Are you using the userscript or ublock variant?

**Comment by Osyruu** [comment]
Edit: I think my browser don't run extensions correctly, my bad.

But if I have problems, I come back to you, Sorry it was a mistake from my part.

**Comment by Osyruu** [comment]
I have chrome
![image](https://user-images.githubusercontent.com/22823858/105013745-dd77fd80-5a3f-11eb-8207-d4cb8c319a02.png)
![image](https://user-images.githubusercontent.com/22823858/105013817-ecf74680-5a3f-11eb-96e3-8de1964724a3.png)

**Comment by pixeltris** [comment]
So what solution are you using exactly? It looks like you have `Twitch AdBlock` enabled. Is that the solution you're trying to use?

To ensure that one is behaving correctly you'll want to open the network inspector in chrome (f12 -> Network tab), refresh a stream and filter for `choosen.dev`. If you don't see that with a `Status` code of 200 then it isn't being applied for some reason.

**Comment by Osyruu** [comment]
![image](https://user-images.githubusercontent.com/22823858/105015413-d3ef9500-5a41-11eb-8a66-4c51a5326752.png)

**Comment by pixeltris** [comment]
If you're using `Twitch AdBlock` and not seeing `choosen.dev` in the network inspector then it's not being applied (or there's a bug in the script). I am not the developer of that extension so I cannot help you beyond that.

**Comment by Osyruu** [comment]
Yeah I don't see `choosen.dev` in the network inspector I don't know why

**Comment by pixeltris** [comment]
In your screenshot you're actually searching in the code. You want to hit click the icon next to the search icon and enter it into the textbox that appears.

**Comment by Osyruu** [comment]
![image](https://user-images.githubusercontent.com/22823858/105017103-b1f71200-5a43-11eb-9cb5-8462d413ddc6.png)

**Comment by pixeltris** [comment]
Yes that's a correct filter. If you're in incognito mode you'll need to go to the extension details and enable it for incognito. If you're not in incognito mode then it should be working, but for some reason isn't working. The developer of that extension is @ChoosenEye it looks like they have a "bugs" repo here which might be a more appropriate place to ask questions related to that extension https://github.com/ChoosenEye/Twitch-Adblock-Bugs

**Comment by Akitakek** [comment]
In my case the ublock solution stopped working today, I had a pre-roll with the
`userResourcesLocation https://raw.githubusercontent.com/pixeltris/TwitchAdSolutions/master/dyn/dyn-ublock-origin.js`

**Comment by pixeltris** [comment]
@Akitakek there was a Twitch API change on the 13th of January. This repo was updated on the 15th to reflect this change. I don't believe the ublock origin variants reload unless you restart your browser (even then it could potentially cache them). See https://github.com/pixeltris/TwitchAdSolutions/issues/12

I just tested preroll/midroll on `dyn` and it's definitely working with the up to date version.
